src/prompt.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+const readline = require('readline');
+
+const prompt = (msg) => {
```

Comment:
I cannot se where you validate this
<img width="285" alt="image" src="https://github.com/mate-academy/node_bulls-and-cows/assets/49764881/92f680c5-160c-4a20-9e25-293e401a009e">